index.js

```diff
@@ -40,6 +40,7 @@ function Peer (opts) {
   self.initiator = opts.initiator || false
   self.channelConfig = opts.channelConfig || Peer.channelConfig
   self.config = opts.config || Peer.config
```

Comment:
What do you think about `Object.assign({}, opts.config, Peer.config)`?
Checking separate properties manually feels very hacky.

Reply:
Sure. Shouldn't it be `Object.assign({}, Peer.config, opts.config)`?

Reply:
Yes, my bad